docs/releases/2.2.txt

```diff
@@ -87,6 +87,9 @@ Minor features
   :class:`~django.contrib.postgres.indexes.SpGistIndex` classes allow
   creating ``B-Tree``, ``hash``, and ``SP-GiST`` indexes in the database.
 
+* The new :class:`~django.contrib.postgres.constraints.ExclusionConstraint`
```

Comment:
move this part to 3.0

Reply:
Yep, done locally. Am working on getting tests and documentation in a better state.